<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compass Password Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="compass.qrc">
    <normaloff>:/icon-128.png</normaloff>:/icon-128.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="categoryView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>17</number>
       </attribute>
      </widget>
      <widget class="QTableView" name="entryView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>17</number>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="searchLabel">
         <property name="text">
          <string>&amp;Search:</string>
         </property>
         <property name="buddy">
          <cstring>searchEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>&amp;Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newDatabaseAction"/>
    <addaction name="openDatabaseAction"/>
    <addaction name="closeDatabaseAction"/>
    <addaction name="separator"/>
    <addaction name="importDatabaseAction"/>
    <addaction name="exportDatabaseAction"/>
    <addaction name="separator"/>
    <addaction name="changeMasterPasswordAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="addEntryAction"/>
    <addaction name="editEntryAction"/>
    <addaction name="removeEntryAction"/>
    <addaction name="separator"/>
    <addaction name="copyUsernameAction"/>
    <addaction name="copyPasswordAction"/>
    <addaction name="separator"/>
    <addaction name="copyEntryAction"/>
    <addaction name="pasteEntryAction"/>
    <addaction name="separator"/>
    <addaction name="addToExportListAction"/>
    <addaction name="removeFromExportListAction"/>
    <addaction name="separator"/>
    <addaction name="addCategoryAction"/>
    <addaction name="editCategoryAction"/>
    <addaction name="removeCategoryAction"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="findInDatabaseAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="optionsAction"/>
    <addaction name="passwordGeneratorAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="newDatabaseAction">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create new database file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="openDatabaseAction">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Opens existing database file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="addEntryAction">
   <property name="text">
    <string>&amp;Add Entry</string>
   </property>
   <property name="toolTip">
    <string>Adds a new entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="removeEntryAction">
   <property name="text">
    <string>&amp;Remove Entry</string>
   </property>
   <property name="toolTip">
    <string>Removes entry</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="editEntryAction">
   <property name="text">
    <string>&amp;Edit/View Entry</string>
   </property>
   <property name="toolTip">
    <string>Edits/Views Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="addCategoryAction">
   <property name="text">
    <string>Add &amp;Category</string>
   </property>
   <property name="toolTip">
    <string>Adds a new category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="removeCategoryAction">
   <property name="text">
    <string>Re&amp;move Category</string>
   </property>
   <property name="toolTip">
    <string>Removes category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="editCategoryAction">
   <property name="text">
    <string>Edit/&amp;View Category</string>
   </property>
   <property name="toolTip">
    <string>Edits/Views category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="findInDatabaseAction">
   <property name="text">
    <string>&amp;Find in Database</string>
   </property>
   <property name="toolTip">
    <string>Finds in Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="copyUsernameAction">
   <property name="text">
    <string>Copy &amp;Username</string>
   </property>
   <property name="toolTip">
    <string>Copy username to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="copyPasswordAction">
   <property name="text">
    <string>Copy &amp;Password</string>
   </property>
   <property name="toolTip">
    <string>Copy password to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="addToExportListAction">
   <property name="text">
    <string>&amp;Add To Export List</string>
   </property>
   <property name="toolTip">
    <string>Adds entry to the export list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="removeFromExportListAction">
   <property name="text">
    <string>&amp;Remove From Export List</string>
   </property>
   <property name="toolTip">
    <string>Removes entry from the export list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="exportDatabaseAction">
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="toolTip">
    <string>Export selected entries to the database file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="importDatabaseAction">
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="toolTip">
    <string>Import database file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="closeDatabaseAction">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close database file</string>
   </property>
  </action>
  <action name="changeMasterPasswordAction">
   <property name="text">
    <string>Change Master &amp;Password</string>
   </property>
   <property name="toolTip">
    <string>Changes master password of database file</string>
   </property>
  </action>
  <action name="addCategoryToExportListAction">
   <property name="text">
    <string>&amp;Add Category To Export List</string>
   </property>
   <property name="toolTip">
    <string>Adds category to the export list</string>
   </property>
  </action>
  <action name="removeCategoryFromExportListAction">
   <property name="text">
    <string>&amp;Remove Category From Export List</string>
   </property>
   <property name="toolTip">
    <string>Removes category from the export list</string>
   </property>
  </action>
  <action name="optionsAction">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="toolTip">
    <string>Program options</string>
   </property>
  </action>
  <action name="passwordGeneratorAction">
   <property name="text">
    <string>&amp;Password Generator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="copyEntryAction">
   <property name="text">
    <string>&amp;Copy Entry</string>
   </property>
   <property name="toolTip">
    <string>Copy entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteEntryAction">
   <property name="text">
    <string>&amp;Paste Entry</string>
   </property>
   <property name="toolTip">
    <string>Paste entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>searchEdit</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>categoryView</tabstop>
  <tabstop>entryView</tabstop>
 </tabstops>
 <resources>
  <include location="compass.qrc"/>
 </resources>
 <connections/>
</ui>
